### DISCUSSION TITLE
p-table seams to scroll my its container

### DISCUSSION THREAD

**Discussion by martin-tauber** [primefaces/community#484] [PrimeNG]
I have a p-table embedded in a div with fixed height (somewhere up the road) which has overflow-auto set. the p-table is quite long and does not fit in the div which caused an overflow. I have some components before the p-table in the div (like title description etc) when I load the page the div is scrolled and not shown from the top. to me it looks like p-table is causing this issue, since when I change the p-table to a long lorem ipsum everything works fine and the div is not scrolled. Any ideas what is causing this?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz reproducer so we can identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
